.github/workflows/cancel.yml

```diff
@@ -0,0 +1,15 @@
+name: Cancel Previous Jobs
+
+on:
+  workflow_call:
+
+jobs:
+  cancel:
+    name: 'Cancel Previous Runs'
```

Comment:
```suggestion
    name: run
```
This is to confirm with the other workflows and how it is rendered in GH Actions GUI.

Reply:
I think `Cancel Previous Runs` was more descriptive than just `run`:
<img width="338" alt="image" src="https://user-images.githubusercontent.com/4221770/157762876-8c41645f-2d51-4608-afe5-d345cd7e5eb0.png">